cylc/flow/scripts/report_timings.py

```diff
@@ -132,7 +132,8 @@ def main(parser: COP, options: 'Values', workflow_id: str) -> None:
         # No output specified - choose summary by default
         options.show_summary = True
 
-    with _get_dao(workflow_id) as dao:
+    db_file = get_workflow_run_pub_db_path(workflow_id)
+    with CylcWorkflowDAO(db_file) as dao:
```

Comment:
Sorry, just spotted something I missed
```suggestion
    with CylcWorkflowDAO(db_file, is_public=True) as dao:
```

Reply:
^